project1/index.html

```diff
@@ -0,0 +1,108 @@
+        rel="stylesheet">
+
+</head>
+
+<body>
+    <hr />
+    <h1>Crane Wife</h1>
+    <hr />
+    <h2><a href="https://cjhauser.com/>"><em>By CJ Hauser</em></a></h2>
+
+    <div class=sub-border>
+        <p class="date"> July 16, 2019</p>
+        <p> Ten days after I called off my engagement I was supposed to go on a scientific expedition to study the
+            whooping
+            crane on the gulf coast of Texas. Surely, I will cancel this trip, I thought, as I shopped for nylon hiking
+            pants
+            that zipped off at the knee. Surely, a person who calls off a wedding is meant to be sitting sadly at home,
+            reflecting on the enormity of what has transpired and not doing whatever it is I am about to be doing that
+            requires
+            a pair of plastic clogs with drainage holes. Surely, I thought, as I tried on a very large and floppy hat
+            featuring
+            a pull cord that fastened beneath my chin, it would be wrong to even be wearing a hat that looks like this
+            when
+            something in my life has gone so terribly wrong.</p>
+
+        <p> Ten days earlier I had cried and I had yelled and I had packed up my dog and driven away from the upstate
+            New
+            York
+            house with two willow trees I had bought with my fiancé.</p>
+
+        <p> Ten days later and I didn’t want to do anything I was supposed to do.</p>
+
+        <div class=star>*</div>
+
+        <p>I went to Texas to study the whooping crane because I was researching a novel. In my novel there were
+            biologists
+            doing field research about birds and I had no idea what field research actually looked like and so the
+            scientists in
+            my novel draft did things like shuffle around great stacks of papers and frown. The good people of the
+            Earthwatch
+            organization assured me I was welcome on the trip and would get to participate in “real science” during my
+            time
+            on
+            the gulf. But as I waited to be picked up by my team in Corpus Christi, I was nervous—I imagined everyone
+            else
+            would
+            be a scientist or a birder and have daunting binoculars.</p>
+
+        <p>The biologist running the trip rolled up in in a large white van with a boat hitch and the words <em>
+                BIOLOGICAL SCIENCES </em>
+            stenciled across the side. Jeff was forty-ish, and wore sunglasses and a backward baseball cap. He had a
+            winter
+            beard and a neon-green cast on his left arm. He’d broken his arm playing hockey with his sons a week before.
+            The
+            first thing Jeff said was, “We’ll head back to camp, but I hope you don’t mind we run by the liquor store
+            first.” I
+            felt more optimistic about my suitability for science.</p>
+
+        <div class=star>*</div>
```

Comment:
Classname should be wrapped in double quotes:
```suggestion
        <div class="star">*</div>
```